server/worker/worker.py

```diff
@@ -12,6 +13,7 @@
     configure_sentry()
     while True:
         run_new_tasks()
+        cleanup_sessions()
```

Comment:
I think the cleanup has to be a background task, since it's based on timeouts, and we can't guarantee there will be a chance to clean it up via normal web requests, because the user may simply not show up again.

It's not a huge deal if this job fails for a few hours, because there are other safeguards preventing the sessions from staying valid beyond their expiration date. This job here is mostly to garbage collect the database periodically.

Reply:
Gotcha, that's good context. I agree a background task is appropriate. I have a hygenic impulse to make it a separate process from the background worker and run it periodically (e.g. using [Heroku Scheduler](https://devcenter.heroku.com/articles/scheduler)), but I can live with piggybacking on the worker process for now.